doc/api.md

```diff
@@ -2,66 +2,6 @@
 
 <!-- TOC -->
 
--   [Preface](#preface)
```

Comment:
Is this removal of the TOC intentional?

Reply:
Restored here: https://github.com/telefonicaid/iotagent-node-lib/pull/1369/commits/a1cb5da739231fb75b1bcdec62c6797e133414a1